docker-compose-services/redis/README.md

```diff
@@ -60,27 +60,70 @@ $ ddev describe
 │          │      │ Host: localhost:54710                   │                       │
 ```
 
-## Examples
+This example shows:
 
-Frameworks, such as Laravel, require configuration to use the redis service.
+* From the webserver point-of-view:
+  * REDIS HOST: ddev-example-redis
+  * REDIS PORT: 6379
+
+* From the host Operating System point-of-view:
+  * REDIS HOST: localhost
+  * REDIS PORT: 54710
+
+## Framework Quickstarts
+
+* Frameworks require configuration to use the redis service.
+* You need to replace `<REDIS_HOST>` values with your project's redis host. Hint: `ddev describe`
+
+## Drupal
+
+* install the [`redis` module](https://www.drupal.org/project/redis)
+
+```shell
+ddev composer require drupal/redis
+```
+
+* enable the module via the website (`/admin/modules`) or via `drush` with the following command
```

Comment:
So you don't *need* to enable the Redis module. This will work happily without it. I would say turning it on enables you to see the connection in the status report and a detailed view at `/admin/reports/redis`